README.md

```diff
@@ -245,6 +245,48 @@ namespace SerilogEventFlow
 }
 ```
 
+#### ILogger
+
+*Nuget package:* **Microsoft.Diagnostics.EventFlow.Inputs.ILogger**
```

Comment:
I is likely that a lot of different libraries will use the ILogger concept and define their own ILogger implementation. For example Serilog has one: https://github.com/serilog/serilog/blob/dev/src/Serilog/ILogger.cs

So I suggest that we name the package and the input so that it conveys it is about Microsoft.Extensions.Logging package and its implementation of ILogger. For example

Microsoft.Diagnostics.EventFlow.Inputs.MicrosoftLogging